Homework2/Task1.py

```diff
@@ -2,4 +2,15 @@
 # и показывает сумму его цифр.
 # Пример:
 # 6782 -> 23
-# 0,56 -> 11
\ No newline at end of file
+# 0,56 -> 11
+
+a = input('Введите число: ')
+n = a
+n = n.replace('.', '')
+n = int(n)
+
+sum = 0
+while (n != 0):
+    sum = sum + n % 10
+    n = n // 10
```

Comment:
```suggestion
while n:
    sum += n % 10
    n //= 10
```
Так ведь проще, не правда ли?